src/situations/SituationOrNoticeIcon.tsx

```diff
@@ -22,6 +22,7 @@ export const SituationOrNoticeIcon = ({
   const situations =
     'situation' in props ? [props.situation] : props.situations;
 
+  // TODO: It might be needed to check if the transport is flexible and shows a yellow icon (warning)
```

Comment:
This is something we need to check if it is possible to add this warning from the OPT planner or import instead of force a warning icon.